templates/networkd_network/networkd_network_generic.j2

```diff
@@ -24,6 +24,9 @@
 {% if item['ipv6_token'] is defined and item['ipv6_token'] != '' %}
 {{ 'IPv6Token=' ~ item['ipv6_token'] }}
 {% endif %}
+{% if item['ipv6_accept_ra'] is defined and item['ipv6_accept_ra'] in ['true', 'false'] %}
+{{ 'IPv6AcceptRA=' ~ item['ipv6_accept_ra'] }}
```

Comment:
The type of variable is `string`. There not needed additional int2str conversion, concat is enough